app/src/main/java/com/android/sample/ui/video/VideoScreen.kt

```diff
@@ -0,0 +1,24 @@
+package com.android.sample.ui.video
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import com.android.sample.ui.navigation.BottomNavigationMenu
+import com.android.sample.ui.navigation.LIST_OF_TOP_LEVEL_DESTINATIONS
+import com.android.sample.ui.navigation.NavigationActions
+
+@Composable
+fun VideoScreen(navigationActions: NavigationActions) {
+  Scaffold(
+      modifier = Modifier.testTag("videoScreen"),
+      content = { padding -> Text("Video Screen", modifier = Modifier.padding(padding)) },
+      bottomBar = {
+        BottomNavigationMenu(
```

Comment:
Yes we could. We could create a composable representing the main navigation bar that we will mostly use, and use default parameters to give him LIST_OF_TOP_LEVEL_DESTINATIONS, and correct behaviour for onTabSelected, but this would require to pass NavigationActions as params. Don't know if it's worth it to save 2 lines. Perhaps, good idea.